docs/designers-developers/developers/block-api/block-edit-save.md

```diff
@@ -6,63 +6,145 @@ When registering a block, the `edit` and `save` functions provide the interface
 
 The `edit` function describes the structure of your block in the context of the editor. This represents what the editor will render when the block is used.
 
+{% codetabs %}
+{% ES5 %}
 ```js
-// Defining the edit interface
+// A static div
 edit() {
-	return <hr />;
+	return wp.element.createElement(
+		'div',
+		null,
+		'Your block.'
+	);
 }
 ```
+{% ESNext %}
+```jsx
+edit() {
+	return <div>Your block.</div>;
+}
+```
+{% end %}
 
 The function receives the following properties through an object argument:
 
 ### attributes
 
-This property surfaces all the available attributes and their corresponding values, as described by the `attributes` property when the block type was registered. In this case, assuming we had defined an attribute of `content` during block registration, we would receive and use that value in our edit function:
+This property surfaces all the available attributes and their corresponding values, as described by the `attributes` property when the block type was registered. See [attributes documentation](/docs/designers-developers/developers/block-api/block-attributes.md) for how to specify attribute sources.
+
+In this case, assuming we had defined an attribute of `content` during block registration, we would receive and use that value in our edit function:
 
+{% codetabs %}
+{% ES5 %}
+```js
+edit( props ) {
+	return wp.element.createElement(
+		'div',
+		null,
+		props.attributes.content
+	);
+}
+```
+{% ESNext %}
 ```js
-// Defining the edit interface
 edit( { attributes } ) {
 	return <div>{ attributes.content }</div>;
 }
 ```
+{% end %}
 
 The value of `attributes.content` will be displayed inside the `div` when inserting the block in the editor.
 
 ### className
 
 This property returns the class name for the wrapper element. This is automatically added in the `save` method, but not on `edit`, as the root element may not correspond to what is _visually_ the main element of the block. You can request it to add it to the correct element in your function.
 
+{% codetabs %}
+{% ES5 %}
+```js
+edit( props ) {
+	return wp.element.createElement(
+		'div',
+		{ className: props.className },
+		props.attributes.content
+	);
+}
+```
+{% ESNext %}
 ```js
-// Defining the edit interface
 edit( { attributes, className } ) {
 	return <div className={ className }>{ attributes.content }</div>;
 }
 ```
+{% end %}
 
 ### isSelected
 
 The isSelected property is an object that communicates whether the block is currently selected.
```

Comment:
I thought `isSelected` was a boolean?

Reply:
Isn't a boolean an object too :-)

Reply:
It's `0`s and `1`s as well! The more specific, the better.